auctex-latexmk.el

```diff
@@ -74,6 +74,10 @@
   "Encoding mapping for platex."
   :group 'auctex-latexmk)
 
+(defcustom auctex-latexmk-inherit-TeX-PDF-mode nil
+  "If non-nil add -pdf flag to latexmk when TeX-PDF-mode is active."
```

Comment:
Please enclose `TeX-PDF-mode` using backquote and singlequote.
It will be helpful in Help mode.